### DISCUSSION TITLE
random reload when clicked on <Link> component

### DISCUSSION THREAD

**Discussion by ghost** [neuefische/questions#471] [Web]
Right now, I am using useAtom as a global state for my array of objects!

Code: 
```
const list = [1,2,3,etc]
export const allObjects = atom(list)
```

Sometimes (seems random) when i navigate through my app and click a <Link> component it triggers a random reload of the page and destroys the current global state!
How to avoid that?

**Comment by doemser** [comment]
There can be many reasons why your global state is adjusted. Please send us an actual PR and/or code examples that show the problem.

**Reply by ghost** [reply]
Thank you for taking a look at it!
It does not happen often, but sometimes :/
Here's my [PR](https://github.com/TariqAlQadi/capstone-project/pull/17)

**Reply by ghost** [reply]
And it ONLY happens when i click on a <Link> component ..

**Reply by ThomasF85** [reply]
I cannot reproduce this error. What component are you clicking on?

**Reply by ghost** [reply]
Right now I can't reproduce it too! 😅 
I will comment if it happens again.
But thanks for taking a look.
